src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rentPrice = 40;
+  const sevenDaysDiscount = 50;
```

Comment:
do not attach var names to days, use abstract names e.g. `smallDiscount` `largeDiscount`